From 67a2616af18bbb38905b90ed1c6b66659671e1da Mon Sep 17 00:00:00 2001
From: Tien Fong Chee <tien.fong.chee@intel.com>
Date: Wed, 20 Jun 2018 15:06:20 +0800
Subject: [PATCH 076/144] common/memsize.c: Increase save array for supporting
 memory size > 4GB

In ARM 64-bits, memory size can be supported is more than 4GB,
hence increasing save array is needed to cope with testing larger memory.

Signed-off-by: Tien Fong Chee <tien.fong.chee@intel.com>
---
 common/memsize.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/memsize.c b/common/memsize.c
index 5670e95137..13b0047786 100644
--- a/common/memsize.c
+++ b/common/memsize.c
@@ -26,7 +26,7 @@ DECLARE_GLOBAL_DATA_PTR;
 long get_ram_size(long *base, long maxsize)
 {
 	volatile long *addr;
-	long           save[31];
+	long           save[BITS_PER_LONG - 1];
 	long           save_base;
 	long           cnt;
 	long           val;
-- 
2.18.0

